src/index.html

```diff
@@ -17,16 +17,28 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search-bar top-search-bar"
```

Comment:
The class names don't seem to represent the meaning of the content. It's better to use class names that describe the content, not the styles or tag names.